NitroxClient/GameLogic/EquipmentSlots.cs

```diff
@@ -10,13 +10,23 @@
 using NitroxModel.Logger;
 using NitroxModel.Packets;
 using NitroxModel_Subnautica.DataStructures;
-using NitroxModel_Subnautica.Helper;
 using UnityEngine;
 
 namespace NitroxClient.GameLogic
 {
     public class EquipmentSlots
     {
+        private List<EquipmentType> ApplicableEquipmentTypes { get; } = new List<EquipmentType>()
```

Comment:
This can't be made readonly it is an unacceptable type in this instance. Unless we want to have a backing field? Though that doesnt seem too logical to me with this.